tests/robot-tests/tests/general_public/fast_track.robot

```diff
@@ -9,19 +9,15 @@ Test Setup          fail test fast if required
 
 
 *** Test Cases ***
-Navigate to publication release page
-    environment variable should be set    PUBLIC_URL
-    user navigates to public frontend    %{PUBLIC_URL}/find-statistics/pupil-absence-in-schools-in-england
-
```

Comment:
We've still deleted this? How would we actually navigate to the right place without this?

Reply:
Ah yeah, good catch. I've added this back in